packages/ia-zendesk-help-widget/src/ia-zendesk-help-widget.js

```diff
@@ -0,0 +1,75 @@
+import { html, LitElement } from 'lit-element';
+import zendeskWidgetCSS from './styles/zendesk-widget';
+import questionIcon from './icon-question-mark';
+
+class ZenDeskHelpWidget extends LitElement {
+  static get styles() {
+    return zendeskWidgetCSS;
+  }
+
+  static get properties() {
+    return {
+      widgetSrc: { type: String },
+      widgetkey: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.widgetSrc = '';
+    this.widgetkey = '';
+  }
+
+  initiateZenDesk() {
+    // Hide custom button
+    this.style.opacity = 0;
+
+    // prevent further if third party script already loaded
+    if (this.abortIfScriptAlreadyLoaded()) return;
+
+    // Inject help widget
+    this.injectScript();
+
+    let countdownTicksLeft = 20;
+    const timeoutTimer = setInterval(() => {
+      // See if button has appeared
+      const iframe = document.getElementById('launcher');
+      if (iframe) {
+        const button = iframe.contentWindow.document.querySelector('button');
+        if (button) {
+          // Found it! Click on it and hope the window opens!
+          button.click();
+          clearInterval(timeoutTimer);
+          return;
+        }
+      }
+
+      // Try again later
+      countdownTicksLeft--;
+      if (countdownTicksLeft <= 0) clearInterval(timeoutTimer);
```

Comment:
We may also want to remove the component entirely from the DOM if we reach this point, just to be certain users navigating via keyboard or screen reader don't find the help button and interact with it.